src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -357,6 +363,8 @@ void InputProcessor::processInput()
     DataIPShortCuts::cNumericFieldNames.allocate(MaxNumeric);
     DataIPShortCuts::rNumericArgs.dimension(MaxNumeric, 0.0);
     DataIPShortCuts::lNumericFieldBlanks.dimension(MaxNumeric, false);
+
+    reportIDFRecordsStats();
```

Comment:
Added call to new function at end of processInput.

Reply:
Perhaps instead we can add it in the OutputReportsTabular or something: only report if "ObjectCountSummary" was included. But it takes little time to run.